src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -794,7 +794,8 @@
         ],
         "Unit Localization": ["TemperatureUnit"],
         "Zone Management": ["TwoDCartesianMax"],
-        "TLS Client Management": ["ClusterRevision", "MaxProvisioned"]
+        "TLS Client Management": ["ClusterRevision", "MaxProvisioned"],
+        "Thermostat": ["CurrentThermostatSuggestion"]
```

Comment:
Since this is a Struct type, no need to list it here?

Reply:
A list works by default. A struct type doesn't. I saw the zap error message where it complains that this was not added.